build-babel/jspm_packages/npm/babel-plugin-transform-regenerator@6.5.2/lib/visit.js

```diff
@@ -250,6 +250,6 @@ var awaitVisitor = {
     // Transforming `await x` to `yield regeneratorRuntime.awrap(x)`
     // causes the argument to be wrapped in such a way that the runtime
     // can distinguish between awaited and merely yielded values.
-    path.replaceWith(t.yieldExpression(t.callExpression(util.runtimeProperty("awrap", state.file.get("regeneratorIdentifier")), [argument]), false));
+    path.replaceWith(t.yieldExpression(t.callExpression(util.runtimeProperty("awrap"), [argument]), false));
   }
 };
```

Comment:
had no idea, those files were replaced automatically by the build scripts ... I thought it was just a new version from babel so I kept them in.